### ISSUE TITLE
Bug: Rancher generates buckets

### ISSUE THREAD

**Issue by tomerorian** [TangoTek/TekTopia-Community#145]
**Describe the bug**
When my rancher milks a cow he has 1 bucket in the inventory which he uses, but then for each cow he just adds a new bucket of milk, keeping the empty one.

**Expected behavior**
The bucket should be replaced with a bucket of milk

**Comment by RensSwag2001** [comment]
![2019-06-01_15 53 39](https://user-images.githubusercontent.com/51168774/58749345-6914d300-8485-11e9-8cde-9562abcde74d.png)
![2019-06-01_15 53 23](https://user-images.githubusercontent.com/51168774/58749349-6ca85a00-8485-11e9-8261-01738e2cb28f.png)

Mine does exact the same!

**Comment by AltSystem42** [comment]
I would like to add to this by saying I only gave my ranches two buckets to start with this is how much milk i ended up with.
![2019-06-21_17 23 29](https://user-images.githubusercontent.com/50642697/59949418-63d5f300-9449-11e9-83e6-f38f1d2aa277.png)

**Comment by LadyGwynn** [comment]
I'm having this issue now too, just since the latest update. My ranchers only had four buckets to start with. I disabled milking on one of my two ranchers after I had to give away a full chestful of milk and I've accumulated almost another chestful since then. The ranchers still have empty buckets and there are empty buckets in my chest as well.
![2019-06-21_17 59 32](https://user-images.githubusercontent.com/42978079/59955323-21171980-944f-11e9-8aec-30bd62e0b313.png)

I'm playing on What If server 1. My coordinates are 8589, 4480.
